pkg/login/social/github_oauth.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"regexp"
 
+	"github.com/grafana/grafana/pkg/models/roletype"
```

Comment:
nit: import ordering - move the external `"golang.org/x/oauth2"` import above the internal dependency and add an empty line.

Reply:
nice! thx